content/pt/docs/tutorials/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Tutoriais
+main_menu: true
+no_list: true
+weight: 60
+content_type: concept
+---
+
+<!-- overview -->
+
+Essa seção da documentação contém tutoriais (em inglês). Um tutorial mostra como realizar um objetivo mais complexo que uma simples [tarefa](/docs/tasks/). Eles podem ser divididos em diversas seções, cada uma com uma sequência de passos e etapas a serem seguidos.
+
+Antes de iniciar um tutorial, é interessante que vocẽ salve a página de [Glossário](/pt/docs/reference/glossary/) para futuras referências.
+
+
+<!-- body -->
+
+## Básicos
+
+* [Tutoriais básicos do Kubernetes](/docs/tutorials/kubernetes-basics/) é um tutorial interativo que auxilia no entendimento do ecossistema Kubernetes, bem como te permite testar algumas funcionalidades básicas do Kubernetes.
```

Comment:
Hi, here the title is on plural and the text on a singular subject. I think the title can be like "Kubernetes básico". Although this tutorial contain any concepts, but the context is the basic of Kubernetes, so should be the title is singular. Thanks